File: day020/classwork/task2.py
# 2) მომხმარებელს შემოატანინეთ რიცხვი, შემდეგ თუ ეს რიცხვი დადებითია დაბეჭდეთ ეს და კიდევ შეამოწმეთ ლუწია თუ კენტი, ხოლო თუ არაა დადებითი მხოლოდ დაბეჭდეთ რომ უარყოფითია
num =int(input("Enter a number: "))
if num > 0:
    print("Positive")
    if num % 2 == 0:
        print("Even")
    else:
        print("Odd")
else:
    print("Negative")

File: day064/homework/task2.js
// 3) შექმენით HTML input <input id="numInput" type="number" placeholder="Enter number"> და ღილაკი <button id="doubleBtn">Double</button>. ასევე პარაგრაფი <p id="numOut"></p>. გამოიყენეთ external JS და ფუნქცია doubleNumber(), რომ ღილაკზე დაჭერისას შეყვანილი რიცხვი ორჯერ გაზრდილი გამოჩნდეს პარაგრაფში
document.getElementById('doubleBtn').onclick = doubleNumber;

function doubleNumber() {
    let num = document.getElementById('numInput').value;
    let doubled = num * 2;
    document.getElementById('numOut').innerText = doubled;
}